src/Actions.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import type { State } from './store';
+import fakeMessages from './messages.json';
+
+type Dispatch = ({type: string}) => void;
+type GetState = () => State;
+
+function fakePromise(data, delay) {
```

Comment:
what's the use for fake data? I thought you were connected to a live data source. please remove all references to a fake source of data.